### ISSUE TITLE
angular.js:14525 TypeError: Cannot read property 'invite' of undefined

### ISSUE THREAD

**Issue by Pusnow** [overleaf/overleaf#684]
## Steps to Reproduce

When I try to share a project with collaborators, it does not work with this error message.

![스크린샷 2020-01-16 오후 2 03 02](https://user-images.githubusercontent.com/6031017/72495123-2fa0a480-3869-11ea-9d4a-0cb649cce4d8.png)

## Expected Behaviour

Successfully adds collaborators.

## Observed Behaviour

Only this mark spinning.

![스크린샷 2020-01-16 오후 2 05 56](https://user-images.githubusercontent.com/6031017/72495187-5232bd80-3869-11ea-82a8-fa20e1747572.png)

## Technical Info

* URL: self-hosetd
* Browser Name and version: Chrome 79.0.3945.117
* Operating System and version (desktop or mobile): mac OS 10.15.2
* Signed in as: admin
* Project and/or file: .

**Comment by mserranom** [comment]
Can you confirm which version of Overleaf CE you're using, and environment variables defined in the compose file?

Particularly [this hack](https://github.com/overleaf/overleaf/issues/652#issuecomment-574658899) could result in the problem you're describing.

**Comment by Pusnow** [comment]
I'm using sharelatex/sharelatex:2.1.0.

environment variables:
```
environment:
    SHARELATEX_APP_NAME: XXXX ShareLaTeX
    SHARELATEX_MONGO_URL: mongodb://mongo/sharelatex
    SHARELATEX_REDIS_HOST: redis
    REDIS_HOST: redis
    

    ENABLED_LINKED_FILE_TYPES: 'url,project_file'
    ENABLE_CONVERSIONS: 'true'
    EMAIL_CONFIRMATION_DISABLED: 'true'

    ## Set for SSL via nginx-proxy
    #VIRTUAL_HOST: latex.xxx.com

    SHARELATEX_SITE_URL: https://latex.xxx.com
    SHARELATEX_NAV_TITLE: XXXX ShareLaTeX Instance
    #SHARELATEX_HEADER_IMAGE_URL: http://somewhere.com/mylogo.png
    SHARELATEX_ADMIN_EMAIL: support@mail.xxx.com

    # SHARELATEX_LEFT_FOOTER: '[{"text": "Powered by <a href=\"https://www.sharelatex.com\">ShareLaTeX</a> 2016"},{"text": "Another page I want to link to can be found <a href=\"here\">here</a>"} ]'
    # SHARELATEX_RIGHT_FOOTER: '[{"text": "Hello I am on the Right"} ]'

    SHARELATEX_EMAIL_FROM_ADDRESS: "sharelatex-noreply@latex.xxx.com"

    # SHARELATEX_EMAIL_AWS_SES_ACCESS_KEY_ID: 
    # SHARELATEX_EMAIL_AWS_SES_SECRET_KEY: 

    SHARELATEX_EMAIL_SMTP_HOST: mail.xxx.com
    SHARELATEX_EMAIL_SMTP_PORT: 25
    SHARELATEX_EMAIL_SMTP_SECURE: 0
    # SHARELATEX_EMAIL_SMTP_USER: 
    # SHARELATEX_EMAIL_SMTP_PASS: 
    # SHARELATEX_EMAIL_SMTP_TLS_REJECT_UNAUTH: true
    # SHARELATEX_EMAIL_SMTP_IGNORE_TLS: false
    # SHARELATEX_CUSTOM_EMAIL_FOOTER: "<div>This system is run by department x </div>"

    ################
    ## Server Pro ##
    ################

    # SANDBOXED_COMPILES: 'true'
    
    # SANDBOXED_COMPILES_SIBLING_CONTAINERS: 'true'
    # SANDBOXED_COMPILES_HOST_DIR: '/var/clsi/compiles'

    ## Works with test LDAP server shown at bottom of docker compose
    # SHARELATEX_LDAP_URL: 'ldap://ldap:389'
    # SHARELATEX_LDAP_SEARCH_BASE: 'ou=people,dc=planetexpress,dc=com'
    # SHARELATEX_LDAP_SEARCH_FILTER: '(uid={{username}})'
    # SHARELATEX_LDAP_BIND_DN: 'cn=admin,dc=planetexpress,dc=com'
    # SHARELATEX_LDAP_BIND_CREDENTIALS: 'GoodNewsEveryone'
    # SHARELATEX_LDAP_EMAIL_ATT: 'mail'
    # SHARELATEX_LDAP_NAME_ATT: 'cn'
    # SHARELATEX_LDAP_LAST_NAME_ATT: 'sn'
    # SHARELATEX_LDAP_UPDATE_USER_DETAILS_ON_LOGIN: 'true'

    # SHARELATEX_TEMPLATES_USER_ID: "578773160210479700917ee5"

    # SHARELATEX_PROXY_LEARN: "true"
```

Also, I've checked `/var/www/sharelatex/web/config/settings.defaults.coffee`, and there is no entry for `recaptcha`.

**Comment by Pusnow** [comment]
It seems to related to https://github.com/overleaf/web/pull/684

**Comment by ZizhenWang** [comment]
I meet the same issue

**Comment by ShaneKilkelly** [comment]
I think this is related to a problem which we're aware of, and are preparing a fix for in the next release cycle: https://github.com/overleaf/overleaf/issues/652

Can you try the temporary fix described in this comment and see if it helps? https://github.com/overleaf/overleaf/issues/652#issuecomment-574658899

Thanks!

**Comment by Pusnow** [comment]
@ShaneKilkelly no helps. There is no `recaptcha`.

This is my `settings.defaults.coffee`. 

[settings.defaults.coffee.txt](https://github.com/overleaf/overleaf/files/4077398/settings.defaults.coffee.txt)

**Comment by mserranom** [comment]
This issue should've been fixed, try running `sharelatex/sharelatex:2.1.1-beta1` image.

**Comment by Pusnow** [comment]
Resolved with `sharelatex/sharelatex:2.1.1-beta1`
